specification/dfp/resource-manager/Microsoft.DFP/preview/2021-02-01-preview/dfp.json

```diff
@@ -0,0 +1,782 @@
+          "description": "The type of the DFP resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the DFP resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Key-value pairs of additional resource provisioning properties."
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "description": "Metadata pertaining to creation and last modification of the resource.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "DFPInstance": {
+      "description": "Represents an instance of a DFP instance resource.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DFPInstanceProperties",
+          "description": "Properties of the provision operation request."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "DFPInstances": {
+      "description": "An array of DFP instance resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DFPInstance"
+          },
+          "description": "An array of DFP instance resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DFPInstanceUpdateParameters": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Key-value pairs of additional provisioning properties."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DFPInstanceMutableProperties",
+          "description": "Properties of the provision operation request"
+        }
+      }
+    },
+    "DFPInstanceProperties": {
+      "description": "Properties of DFP resource.",
+      "type": "object",
+      "properties": {
+        "state": {
+          "type": "string",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Paused",
+            "Suspended",
+            "Provisioning",
+            "Updating",
+            "Suspending",
+            "Pausing",
+            "Resuming",
+            "Preparing",
+            "Scaling"
+          ],
+          "x-ms-enum": {
+            "name": "State",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current state of DFP resource. The state is to indicate more states outside of resource provisioning."
```

Comment:
>more states outside of resource provisioning. [](start = 86, length = 45)

can you clarify what states these are? Since today this seems to be a duplicate of provisioningState

Reply:
I think it is an error. FIxed